barretenberg/cpp/src/barretenberg/commitment_schemes/small_subgroup_ipa/small_subgroup_ipa_utils.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "barretenberg/common/ref_array.hpp"
+#include "barretenberg/constants.hpp"
+
+#include <array>
+#include <string>
+
+namespace bb {
+
+/**
+ * @brief Shared by Prover and Verifier. `label_prefix` is either `Libra:` or `Translation:`.
+ */
+inline std::array<std::string, NUM_SMALL_IPA_EVALUATIONS> get_evaluation_labels(const std::string& label_prefix)
```

Comment:
I'm a little bit confused by some of the ordering in the comments vs code in this file. Maybe the code on the first two methods is consistent but the comment on the second method should be `G(r), A(g * r), A(r), Q(r)`?

Reply:
Or is this just a nomenclature issue and we're calling `A(r)` the shifted eval? I suppose its all relative..